### ISSUE TITLE
Group descriptions not converted

### ISSUE THREAD

**Issue by mattether** [kminami/apib2swagger#27]
Using the following

```
FORMAT: 1A
HOST: https://example.com

# Example

Intro


# Group Test

Group description.

## /endpoint

### GET [GET]

Endpoint description.

+ Response 200
```

The group description does not get converted

Output:

```
swagger: '2.0'
info:
  title: Example
  version: ''
  description: Intro
host: example.com
basePath: /
schemes:
  - https
paths:
  /endpoint:
    get:
      responses:
        '200':
          description: OK
          headers: {}
          examples: {}
      summary: GET
      description: Endpoint description.
      tags:
        - Test
      parameters: []
definitions: {}
```

**Comment by kminami** [comment]
Sorry for late response.
Swagger doesn’t have group level.
Is it good for you to merge group description to endpoint description?

**Comment by kminami** [comment]
I noticed that swagger.tags can be used to store group descriptions.
Implemented and released as v1.3.0.
Thank you!
